src/components/AutoSuggest/index.tsx

```diff
@@ -92,6 +94,34 @@ const AutoSuggest = ({
     // Prevent the text input field(which is the field visible in the UI) from blurring if "focused" is set to "true"
     const handleBlur = useCallback(() => focused && textInputFieldRef.current?.focus(), [focused]);
 
+    const keyboardNavigation = (e: React.KeyboardEvent<HTMLSpanElement> | React.KeyboardEvent<SVGSVGElement>) => {
+        if (suggestions && listRef?.current) {
+            if (e.key === 'ArrowDown' && curr < suggestions.length - 1) {
+                setCurr(prev => prev + 1);
+                // Scroll down after first three options
+                if (curr >= 2) {
+                    listRef.current.scrollBy(0, 40);
```

Comment:
I think we should create a constant for this `40`. I'm guessing this is the height of one item in the dropdown.

Reply:
yes, it's the height of one item